src/main/cpp/VideoState.cpp

```diff
@@ -31,47 +30,23 @@ int VideoState::stream_component_open(int stream_index) {
 	switch (avctx->codec_type) {
 		case AVMEDIA_TYPE_AUDIO:
 			last_audio_stream = stream_index;
-			forced_codec_name = audio_codec_name;
 			break;
 		case AVMEDIA_TYPE_SUBTITLE:
 			last_subtitle_stream = stream_index;
-			forced_codec_name = subtitle_codec_name;
 			break;
 		case AVMEDIA_TYPE_VIDEO:
 			last_video_stream = stream_index;
-			forced_codec_name = video_codec_name;
 			break;
 	}
 
-	if (forced_codec_name)
-		codec = avcodec_find_decoder_by_name(forced_codec_name);
-	if (!codec) {
-		if (forced_codec_name) {
-			av_log(NULL, AV_LOG_WARNING, "No codec could be found with name '%s'\n", forced_codec_name);
-		} 
-		else {
-			av_log(NULL, AV_LOG_WARNING, "No decoder could be found for codec %s\n", avcodec_get_name(avctx->codec_id));
-		}
-		ret = AVERROR(EINVAL);
-		goto fail;
-	}
-
 	avctx->codec_id = codec->id;
-	if (stream_lowres > codec->max_lowres) {
-		av_log(avctx, AV_LOG_WARNING, "The maximum value for lowres supported by the decoder is %d\n",
-			codec->max_lowres);
-		stream_lowres = codec->max_lowres;
-	}
-	avctx->lowres = stream_lowres;
 
-	if (fast)
+	if (ENABLE_FAST_DECODE)
```

Comment:
Same here (see VideoState.h comment on line 75)

Reply:
Changed to use const